### PR TITLE
Add a waffle flag for invalidation testing.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/groups/views.py [hunk 1]
```diff
@@ -576,3 +578,30 @@ def send_invitation_email(request, invite_pk):
...
+
+@never_cache
+@waffle_flag('force_group_invalidation', '404')
+def force_group_invalidation(request, url, alias_model, template=''):
+    """View to help test different scenarios.
+
+    Forces an immediate invalidation in the case that a group is
+    set to expire its memberships.
+    """
+    group_alias = get_object_or_404(alias_model, url=url)
```
Comment (johngian) [inline mozillians/groups/views.py]: Why don't you use group.name directly?

File: mozillians/groups/views.py [hunk 2]
```diff
@@ -576,3 +578,30 @@ def send_invitation_email(request, invite_pk):
...
+    group = group_alias.alias
+    is_curator = request.user.userprofile in group.curators.all()
```
Comment (johngian) [inline mozillians/groups/views.py]: nit: I would prefer ``group.curators.filter()``

File: mozillians/groups/views.py (hunk 1 above)
Comment (akatsoulas) [inline mozillians/groups/views.py]: We need the alias model first in order to distinguish `groups` from `skills`.

**Comment by akatsoulas** [comment]
@johngian pr updated

**Pull request merged**